manifests/katello.pp

```diff
@@ -53,6 +53,7 @@
     # (especially in the custom certs scenario)
     files            => ["${katello_rhsm_setup_script_location}:755=${katello_www_pub_dir}/${katello_rhsm_setup_script}"],
     bootstrap_script => inline_template('/bin/bash <%= @katello_rhsm_setup_script_location %>'),
+    postun_script    => 'test -f /etc/rhsm/rhsm.conf.kat-backup && command cp /etc/rhsm/rhsm.conf.kat-backup /etc/rhsm/rhsm.conf.kat-backup',
```

Comment:
There is a possibility to get this output:

```
test -f /etc/rhsm/rhsm.conf.kat-backup && command cp /etc/rhsm/rhsm.conf.kat-backup /etc/rhsm/rhsm.conf.kat-backup
cp: ‘/etc/rhsm/rhsm.conf.kat-backup’ and ‘/etc/rhsm/rhsm.conf.kat-backup’ are the same file
```

Which exits `1` -- should we prefer to just force copy?